pkg/terraform/terraform.go

```diff
@@ -111,13 +113,143 @@ type TerraformEntrypoint struct {
 	BackendFile string
 }
 
+type ModuleUsageGraph struct {
+	EntrypointToModules  map[string]map[string]struct{}
```

Comment:
Might be very helpful to add comments here explaining what the inner and outer keys are in these maps (module names?)

Reply:
Done